### ISSUE TITLE
IS-04-01: Ensure that changes to resources are reflected in the registry

### ISSUE THREAD

**Issue by andrewbonney** [AMWA-TV/nmos-testing#345]
A common error for some implementations has been that when a resource changes, it updates in the Node API but not in the registry. I don't believe we currently have any tests that ensure when a resource is updated the change is reflected into the registry.

**Comment by garethsb-sony** [comment]
Agree! Though since Node API is almost entirely read-only we only have /target or Connection API to explicitly drive this.

**Comment by garethsb** [comment]
See #604 for a report of this issue in an implementation.

**Comment by garethsb** [comment]
@lo-simon

Updating `IS0401Test.test_13` and `test_14` to check that they bump the resource `version` would be a start, then that the Registry is also updated.

Since the `/target` API that those test cases check is deprecated, it would also be good to extend IS-05-02 to provide a mock Registry and check the same things after using the Connection API.
